src/webui/www/private/scripts/client.js

```diff
@@ -685,17 +688,41 @@ window.addEventListener("DOMContentLoaded", (event) => {
             const span = trackerFilterItem.firstElementChild;
             span.lastChild.textContent = `${text} (${count})`;
 
+            switch(host) {
+                case TRACKERS_ANNOUNCE_ERROR:
+                case TRACKERS_ERROR:
+                    span.lastElementChild.src = "images/tracker-error.svg";
+                    break;
+                case TRACKERS_TRACKERLESS:
+                    span.lastElementChild.src = "images/trackerless.svg";
+                    break;
+                case TRACKERS_WARNING:
+                    span.lastElementChild.src = "images/tracker-warning.svg";
+                    break;
+            }
+
             return trackerFilterItem;
         };
 
-        let trackerlessTorrentsCount = 0;
-        for (const { full_data: { trackers_count: trackersCount } } of torrentsTable.getRowValues()) {
-            if (trackersCount === 0)
-                trackerlessTorrentsCount += 1;
+        let trackerless_count = 0;
+        let has_tracker_error_count = 0;
+        let has_announce_error_count = 0;
+        let has_tracker_warning_count = 0;
```

Comment:
The `has_` prefix feels weird. I would suggest stripping them.
```suggestion
        let trackerErrorCount = 0;
        let trackerAnnounceErrorCount = 0;
        let trackerWarningCount = 0;
```

Reply:
`has_tracker_error` is how the attribute on the API response is named. that's why I kept it the same way for easy grepability. I would like to keep it that way